src/azul/plugins/metadata/hca/transform.py

```diff
@@ -560,6 +560,18 @@ def _specimen(self, specimen: api.SpecimenFromOrganism) -> MutableJSON:
             '_type': 'specimen'
         }
 
+    def _sample_specimen(self, entity: JSON) -> MutableJSON:
```

Comment:
I agree that the placement of these may not be an improvement. The _sample_foo method refers to keys defined in _foo and _sample so they could be placed in proximity to either.